workos/mfa.py

```diff
@@ -88,6 +89,59 @@ def enroll_factor(
 
         return WorkOSAuthenticationFactorSms.construct_from_response(response).to_dict()
 
+    def get_factor(
+        self,
+        id=None,
```

Comment:
Can we rename this `authentication_factor_id` to match the other methods? (also applies to the `delete_factor` method)

Reply:
Updated